internal_transfer_with_agreed_amount/i18n/es.po

```diff
@@ -0,0 +1,120 @@
+msgid "Agreed Amount"
+msgstr "Monto Acordado"
+
+#. module: internal_transfer_with_agreed_amount
+#: model:ir.model.fields,help:internal_transfer_with_agreed_amount.field_internal_transfer_multicurrency__agreed_amount
+msgid "Agreed amount in the currency of the destination bank."
+msgstr "Cantidad acordada en la moneda del banco de destino."
+
+#. module: internal_transfer_with_agreed_amount
+#: model:res.groups,name:internal_transfer_with_agreed_amount.allow_confirm_internal_transfer_with_agreed_amount
+msgid "Allow to confirm an internal transfer with an agreed amount"
+msgstr "Permitir confirmar una transferencia interna con un monto acordado"
+
+#. module: internal_transfer_with_agreed_amount
+#: model_terms:ir.ui.view,arch_db:internal_transfer_with_agreed_amount.view_internal_transfer_multicurrency
+msgid "Apply"
+msgstr "Aplicar"
+
+#. module: internal_transfer_with_agreed_amount
+#: model_terms:ir.ui.view,arch_db:internal_transfer_with_agreed_amount.view_internal_transfer_multicurrency
+msgid "Cancel"
+msgstr "Cancelar"
+
+#. module: internal_transfer_with_agreed_amount
+#: model_terms:ir.ui.view,arch_db:internal_transfer_with_agreed_amount.view_account_payment_form_inherit
+msgid "Confirm with an agreed amount"
+msgstr "Confirmar con un monto acordado"
+
+#. module: internal_transfer_with_agreed_amount
+#: model:ir.model.fields,field_description:internal_transfer_with_agreed_amount.field_internal_transfer_multicurrency__create_uid
+msgid "Created by"
+msgstr "Creado por"
+
+#. module: internal_transfer_with_agreed_amount
+#: model:ir.model.fields,field_description:internal_transfer_with_agreed_amount.field_internal_transfer_multicurrency__create_date
+msgid "Created on"
+msgstr "Creado el"
+
+#. module: internal_transfer_with_agreed_amount
+#: model:ir.model.fields,field_description:internal_transfer_with_agreed_amount.field_internal_transfer_multicurrency__currency_id
+msgid "Currency"
+msgstr "Moneda"
+
+#. module: internal_transfer_with_agreed_amount
+#: model:ir.model.fields,field_description:internal_transfer_with_agreed_amount.field_internal_transfer_multicurrency__display_name
+msgid "Display Name"
+msgstr "Nombre mostrado"
+
+#. module: internal_transfer_with_agreed_amount
+#: model_terms:ir.ui.view,arch_db:internal_transfer_with_agreed_amount.view_account_payment_form_inherit
+msgid "Edit agreed amount"
+msgstr "Editar monto acordado"
+
+#. module: internal_transfer_with_agreed_amount
+#: model_terms:ir.ui.view,arch_db:internal_transfer_with_agreed_amount.view_internal_transfer_multicurrency
+msgid "Fill with the total amount that you will receive for this operation"
+msgstr "Complete con el monto total que recibirá por esta operación"
+
+#. module: internal_transfer_with_agreed_amount
+#: model:ir.model.fields,field_description:internal_transfer_with_agreed_amount.field_internal_transfer_multicurrency__id
+msgid "ID"
+msgstr "ID"
```

Comment:
Did you add this manually?

When te translation is the same as the original term, it shouldn't be provided, because when exporting, it will be cleared anyway.

Reply:
Ok, I already applied the correction.